main.tex

```diff
@@ -15,35 +15,33 @@
+  Despite the potential for private use, research has shown that users' activity can often be traced in practice.
+  This lack of inherent privacy has given rise to business models built on dragnet surveillance of Bitcoin users which is problematic for Bitcoin's fungibility.
 
-% TODO citations should not be included in the abstract, the abstract should be cleaned up, and the remaining information and citations split off into an introduction section with citations
+  A number of methods have been proposed to mitigate these issues. Among these, CoinJoin is an approach to structuring transactions which adds ambiguity and breaks some common assumptions that underlie heuristics used for surveillance.
+  Current implementations of CoinJoin transactions present a number of limitations which may partly explain their lack of widespread adoption.
 
-Chaumian CoinJoin~\cite{mizrahi2013blind}\cite{maxwell2013coinjoin} is a technique to facilitate untrusted construction of collaborative Bitcoin transactions, also known as CoinJoins, by utilizing Chaumian blind signatures~\cite{chaum1983blind}. However this technique requires standard denominations, which limits how such transactions can be constructed.
+  This work introduces a new protocol for centrally coordinated CoinJoin implementations utilizing keyed verification anonymous credentials and homomorphic value commitments. This is an improvement on the state of the art in both privacy and flexibility, enabling some additional use cases and reduced overheads.
+\end{abstract}
 
-We propose to switch to a Keyed-Verification Anonymous Credentials-based (KVAC) scheme~\cite{chase2019signal} to enable more flexible transaction styles, like SharedCoin and CashFusion~\cite{cashfusion} style transactions and Knapsack~\cite{maurer2017anonymous} mixing. Our generalization also enables consolidation of UTXOs, minimizing unmixed change, relaxing minimum required denominations, payments in CoinJoins, better block space efficiency, and PayJoins in CoinJoins. We call this new protocol: WabiSabi.
+\section{Introduction}
 
-\end{abstract}
+Bitcoin transactions redistribute funds by consuming as inputs the outputs of previous transactions, and creating new outputs. The protocol rules enforced by the network ensure that transactions do not arbitrarily inflate the monetary supply and that output are spent at most once. This can be simply verified because the amounts as well as the specific outputs being spent are broadcast as part of the transaction in the clear. This presents significant challenges for transacting with privacy\footnote{In this work we restrict the discussion of Bitcoin privacy to that of transactions, but there are other considerations especially at the network layer.}. % TODO cite Ron, Fistful of Bitcoins, Moser, ...?
```

Comment:
```suggestion
Bitcoin transactions redistribute funds by consuming outputs of previous transactions as inputs, and creating new outputs. The protocol rules enforced by the network ensure that transactions do not arbitrarily inflate the monetary supply and that outputs are spent at most once. This can be verified because the amounts as well as the specific outputs being spent are broadcast as part of the transaction in the clear. This presents significant challenges for transacting with privacy\footnote{In this work we restrict the discussion of Bitcoin privacy to that of transactions, but there are other considerations especially at the network layer.}. % TODO cite Ron, Fistful of Bitcoins, Moser, ...?
```

Reply:
"simply" was added to contrast with e.g. cryptocurrencies where the verification is not simple, since strictly speaking it could be verified otherwise, i will rephrase so that it doesn't look like a fluff word

Reply:
[Don't use simply.](https://jameshfisher.com/2017/02/22/dont-use-simply/)

> Uses of “simply” are an attempt to glorify the writer instead of comfort the reader. The subtext is “I the writer am smart. I find this simple. You should too. If you don’t, you’re stupid.”

Reply:
> While some newer cryptocurrencies use more sophisticated approaches to defining such rules, in Bitcoin the amounts as well as the specific outputs being spent are broadcast in the clear as part of the transaction. This presents significant challenges for transacting with privacy

how's this?

Reply:
also for the record, while i agree with that post in general, this was not the same usage of "simply", it has nothing to do with the reader, only with the fact that in bitcoin these rules are simple because they don't use fancy crypto